lighty-modules/lighty-gnmi/lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/GnmiWithoutRestconfTest.java

```diff
@@ -193,7 +193,7 @@ public static void tearDown() {
         boolean successfullyClosedResources = true;
         gnmiDevice.stop();
         try {
-            Boolean closedSbMod = gnmiSouthboundModule.shutdown().get(TIMEOUT_MILLIS,  TimeUnit.MILLISECONDS);
+            Boolean closedSbMod = gnmiSouthboundModule.shutdown(TIMEOUT_MILLIS,  TimeUnit.MILLISECONDS);
```

Comment:
Please rework this method. Remove Try/catch blocks and check `if (!closedSbMod) {`
And delete `@SuppressWarnings("checkstyle:illegalCatch")` which is not required without `} catch (Exception e) {`

Reply:
method reworked and annotation removed